Improve markdown explanation (#151)
dodona-edu/dodona-edu.github.io: en/references/exercise-description/README.md

## Markdown

> For Dodona exercises and reading activities, we recommend using Markdown over HTML. Descriptions for courses and series can also be written in Markdown. Markdown is a markup language like HTML, but a lot simpler. Below, you can find more information about the basic formatting and a list of all syntax in this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). The rest of this page contains specific extentions (in both Markdown and HTML) that are available in exercise descriptions on Dodona.

Reviewer: ```suggestion
For Dodona exercises and reading activities, we recommend using Markdown over HTML. Descriptions for courses and series can also be written in Markdown. Markdown is a markup language like HTML, but a lot simpler. Below, you can find more information about the basic formatting and a list of all syntax in this [Markdown Cheatsheet page](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). The rest of this page contains specific extensions (in both Markdown and HTML) that are available in exercise descriptions on Dodona.
```
(https://www.askdifference.com/extention-vs-extension/)

### Headers

> In markdown, it is easy to use headers of different levels. For this, you simply put one or more hash symbols (`#`) in front of your title. Depending on the number of symbols used, a larger or smaller title will be shown.

Reviewer 2: ```suggestion
In Markdown, it is easy to use headers of different levels. For this, you simply put one or more hash symbols (`#`) in front of your title. Depending on the number of symbols used, a larger or smaller title will be shown.
```

I think we should consistently capitalise Markdown (or not), but most other occurrences seem to be capitalized (

```markdown
# Level 1
## Level 2
### Level 3
#### Level 4
##### Level 5
###### Level 6
```

### Text formatting

> For simple text formatting, place tekst between stars (`*`). If one or more words are between single starts, the text will render in italics, if you use double stars, it wil render in bold. If you place words between double tilde signs (`~`), the text will be striked through. To format text as code, place the text between backticks (`` ` ``).

Reviewer 2: ```suggestion
For simple text formatting, place tekst between stars (`*`). If one or more words are between single starts, the text will render in italics, if you use double stars, it wil render in bold. If you place words between double tilde signs (`~`), the text will be struck through. To format text as code, place the text between backticks (`` ` ``).
```

I think the verb is `struck through`, while a text that has that style applied is called a `strikethrough` (according to wikidictionary at least https://en.wiktionary.org/wiki/strikethrough#English)

```markdown
This is an example with words in *italics*, **bold**, and ~~striked through~~ and with a variable `a` in function `returnSolution()`.
```

> This example results in this formatting: This is an example with words in *italics*, **bold**, and ~~striked through~~ and with a variable `a` in function `returnSolution()`.

Reviewer 2: ```suggestion
This example results in this formatting: This is an example with words in *italics*, **bold**, and ~~strikethrough~~ and with a variable `a` in function `returnSolution()`.
```

### Hyperlinks and images

> Hyperlinks and images are a little more complex. They consist out of two parts of which you place the first part between square brackets (`[]`) and the second part between round pbrackets (`()`). The first part is used for the words that need to be linked or the image description, the second part is used for the url of the page or image. To indicate you want to render an image, place an exclamation mark (`!`) in front of the first square bracket.

Reviewer 2: ```suggestion
Hyperlinks and images are a little more complex. They consist out of two parts of which you place the first part between square brackets (`[]`) and the second part between round brackets (`()`). The first part is used for the words that need to be linked or the image description, the second part is used for the URL of the page or image. To indicate you want to render an image, place an exclamation mark (`!`) in front of the first square bracket.
```

(Not sure about the URL/url, but I think it is often capitalised in English)
